presentation/src/main/java/com/android254/presentation/sessions/components/SessionStateComponent.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.presentation.sessions.components
+
+import android.os.Build
+
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android254.droidconKE2023.presentation.R
+import com.android254.presentation.common.components.SessionsCard
+import com.android254.presentation.common.components.SessionsLoadingSkeleton
+import com.android254.presentation.models.SessionPresentationModel
+import com.android254.presentation.sessions.models.SessionsUiState
+import com.droidconke.chai.atoms.ChaiDarkGrey
+import com.droidconke.chai.atoms.MontserratRegular
+import com.google.accompanist.swiperefresh.SwipeRefresh
+import com.google.accompanist.swiperefresh.SwipeRefreshState
+import com.google.accompanist.swiperefresh.rememberSwipeRefreshState
+import java.util.*
+
+@RequiresApi(Build.VERSION_CODES.O)
```

Comment:
We had a PR that removed this, you should also remove them

Reply:
I've resolved this.